src/services/actions/orderDetails.ts

```diff
@@ -52,4 +49,12 @@ export function postOrder(ingredients: string[], authToken: string) {
         })
       }))
   }
+}
+
+export function clearConstructor() {
+  return function (dispatch: Dispatch) {
+    dispatch({
+      type: CLEAR_CONSTRUCTOR
+    })
+  }
 }
```

Comment:
This looks over-engineered. It's usually done through action creators like this:
```ts
export function clearConstructorAction() {
  return {
      type: CLEAR_CONSTRUCTOR
    }
}
```

Then used as `dispatch(clearConstructorAction())`. 

Looks like many of your action creators can be simplified this way.

Reply:
Looks like many of your actions, those that don't need thunks at least, can be simplified this way. But it's for another PR